.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,40 @@
+# See https://github.com/golangci/golangci-lint-action for more information
+name: golangci-lint

```

Comment:
I thought we may have already be doing this somewhere else. Either as part of some other meta linter or just this. It could be that the cache just needs to be reset on the other job?

Reply:
It might be the cache, I'll keep reading

Reply:
You were right, we are running it with https://github.com/github/super-linter